soc/ti/k3/j721e/Kconfig.soc

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_J721E
+	bool "TI Jacinto J721E SoC series"
+	select SOC_FAMILY_TI_K3
+	help
+	  Enable support for the TI Jacinto J721E SoC series
+
+config SOC_SERIES_J721E_A72
+	bool
+	select SOC_SERIES_J721E
+	help
+	  Enable support for TI K3 J721e A72 Series
+
+config SOC_SERIES_J721E_R5
+	bool
+	select SOC_SERIES_J721E
+	help
+	  Enable support for TI K3 J721e R5 Series
```

Comment:
these symbols to go, they are not series

Reply:
Sure, removed.